release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -957,11 +964,41 @@ module openconfig-if-ip {
     }
 
     leaf suppress {
+      status deprecated;
       type boolean;
       default false;
       description
-        "When set to true, router advertisement neighbor discovery
-        messages are not transmitted on this interface.";
+        "When set to true, router advertisement neighbor discovery messages
+        are not transmitted on this interface.  Responses to router
+        solicitation are transmitted.";
+    }
+
+    leaf mode {
+      type enumeration {
+        enum ENABLE {
+          description
+            "The system should not respond to ARP requests that
+            do not specify an IP address configured on the local
+            subinterface as the target address.";
+        }
+        enum DISABLE {
+          description
+            "All ipv6 router advertisement functions are disabled.  The
+            local system will not transmit neighbor discovery messages and
+            will not respond to router solicitation messages.";
+        }
+        enum DISABLE_RA {
+          description
+            "Router advertisement neighbor discovery messages are not
```

Comment:
Would be good to add `Unsolicited` router advertisement ..., since responses to RS also use RA message format